com.unity.netcode.adapter.utp/Runtime/com.unity.netcode.adapter.utp.asmdef

```diff
@@ -15,6 +15,12 @@
     "precompiledReferences": [],
     "autoReferenced": true,
     "defineConstraints": [],
-    "versionDefines": [],
+    "versionDefines": [
+        {
+            "name": "com.unity.multiplayer.tools",
+            "expression": "",
+            "define": "MULTIPLAYER_TOOLS"
```

Comment:
Could we call this define something like `UNITY_MULTIPLAYER_TOOLS_PACKAGE` or `UNITY_MP_TOOLS_PACKAGE` so that we're less likely to clash with a user's preprocessor definition?

I worry otherwise what might happen if users had their own in-house multiplayer tools and happen to use this same preprocessor name.

Reply:
This is what we have currently in NGO. It's a good point though. I'll bring it up. For now I'll leave this as is.